packages/prisma/schema.prisma

```diff
@@ -166,6 +166,8 @@ model DestinationCalendar {
   id           Int         @id @default(autoincrement())
   integration  String
   externalId   String
+  /// @zod.email()
+  primaryEmail String?
```

Comment:
This new field is required because if user has selected any secondary calendar as destination calendar then the externalId is no an actual email. (for eg:- it of format fafasdfsfsaffasdf@group.calendar.google.com )